packages/babel-preset-react-app/package.json

```diff
@@ -12,19 +12,20 @@
     "dependencies.js"
   ],
   "dependencies": {
-    "@babel/core": "7.0.0-beta.38",
-    "@babel/plugin-proposal-class-properties": "7.0.0-beta.38",
-    "@babel/plugin-syntax-dynamic-import": "7.0.0-beta.38",
-    "@babel/plugin-transform-classes": "7.0.0-beta.38",
-    "@babel/plugin-transform-destructuring": "7.0.0-beta.38",
-    "@babel/plugin-transform-react-constant-elements": "7.0.0-beta.38",
-    "@babel/plugin-transform-react-display-name": "7.0.0-beta.38",
-    "@babel/plugin-transform-react-jsx": "7.0.0-beta.38",
-    "@babel/plugin-transform-regenerator": "7.0.0-beta.38",
-    "@babel/plugin-transform-runtime": "7.0.0-beta.38",
-    "@babel/preset-env": "7.0.0-beta.38",
-    "@babel/preset-flow": "7.0.0-beta.38",
-    "@babel/preset-react": "7.0.0-beta.38",
+    "@babel/core": "7.0.0-beta.41",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.41",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.41",
+    "@babel/plugin-syntax-dynamic-import": "7.0.0-beta.41",
+    "@babel/plugin-transform-classes": "7.0.0-beta.41",
+    "@babel/plugin-transform-destructuring": "7.0.0-beta.41",
+    "@babel/plugin-transform-react-constant-elements": "7.0.0-beta.41",
+    "@babel/plugin-transform-react-display-name": "7.0.0-beta.41",
+    "@babel/plugin-transform-react-jsx": "7.0.0-beta.41",
```

Comment:
Just curious, is there a reason you bump this dependency up to 41 if you remove the plugin from packages/babel-preset-react-app/index.js?

Reply:
Nah, just oversight. Will fix and update to beta.42 (which will fix the .slice() error and give a better error message) in a bit.

Reply:
Great - Thanks!! 😄